### ISSUE TITLE
Mentor Page page accessibility issues

### ISSUE THREAD

**Issue by gixxerblade** [Vets-Who-Code/vets-who-code-app#222]
# Accessibility

## ARIA IDs are not unique 

Each ID in your HTML document must be unique. [Learn More](https://web.dev/duplicate-id-aria/?utm_source=lighthouse&utm_medium=devtools)

### Failing Element

- Technical: Mentors will assist students in understanding the technical aspects …`<div role="button" id="heading-0" class="panel-heading" tabindex="0">`

- Professional: Mentors will help mentees understand the professional aspects of …`<div role="button" id="heading-1" class="panel-heading" tabindex="0">`

## Form elements do not have associated labels

Labels ensure that form controls are announced properly by assistive technologies, like screen readers. [Learn More](https://web.dev/label/?utm_source=lighthouse&utm_medium=devtools)

### Failing Elements

- `<input type="text" class="form-control input-lg" placeholder="Name" name="name" id="name">`
- `<input type="email" class="form-control input-lg" id="email" placeholder="Email" name="email">`
- `<input type="text" class="form-control input-lg" id="branch-of-service" placeholder="Thank you for your service" name="branch-of-service">`
- `<input type="text" class="form-control input-lg" id="technical-expertise" placeholder="What languages you write in?" name="technical-expertise">`
- `<input type="text" class="form-control input-lg" id="github-portfolio-or-linkedin" placeholder="Share your work" name="github-portfolio-or-linkedin">`
- `<input type="text" class="form-control input-lg" id="location" placeholder="Location" name="location">`
- `<textarea class="form-control" rows="7" placeholder="Please put here any employer restrictions about writing and reading code?" id="employer-restrictions" name="employer-restrictions"></textarea>`

**To Reproduce**
1. Go to 'https://vetswhocode.io/mentor'
2. Click on 'Developer Tools (F12)'
3. Click on Lighthouse
4. Generate report

**Expected behavior**
-  ARIA IDs are not unique - Each ID in theHTML document must be unique. Using the same ID on more than one element may cause screen readers and other assistive technologies to only announce the first element with the shared ID, preventing users from accessing the later elements.
- Form elements do not have associated labels - Labels ensure that form controls are announced properly by assistive technologies like screen readers. Assistive technology users rely on these labels to navigate forms. Mouse and touchscreen users also benefit from labels because the label text makes a larger click target.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by gixxerblade** [comment]
I'll take this one
